microprofile/galleon-common/src/main/resources/layers/standalone/open-tracing/layer-spec.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="open-tracing">
```

Comment:
I think inverting this is more useful (i.e. microprofile-opentracing depends open-tracing, with open-tracing having the features). In any case where the user adds either, it works. If anyone uses one of our aggregate layers, like microprofile-platform or observability, if they exclude either one it works.

The microprofile-opentracing would have to optionally depend open-tracing though; otherwise you could not exclude open-tracing.

In WF 24 we could drop open-tracing.

Reply:
ok